client/src/components/NewPastaForm.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+
+const NewPastaForm = ({ postPasta }) => {
+    const [newPasta, setNewPasta] = useState({
+    name: "",
+    description: "",
+  });
+
+  const handleInputChange = event => {
+    setNewPasta({
+      ...newPasta,
+      [event.currentTarget.name]: event.currentTarget.value
+    });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    newPasta.name.toLowerCase()
```

Comment:
I'd recommend any type of formatting like this to be handled on the back end where we are checking other things about the data as well, like with clean user input. You can ensure it is lowercase before passing the data to be inserted